dev/src/commands/AbstractGenerateStubCommand.ts

```diff
@@ -48,10 +58,9 @@ export default abstract class AbstractGenerateStubCommand extends AbstractGenera
         this.selectedDefinition = "";
         this.fqPathToDefinition = "";
         this.fqPathOutputLocation = "";
-        this.childSourceFolderToAppend = "";
 
         var currentGeneratorTypes: string [] = [];
-        var preferredSourceLocation: string;
+        var preferredSourceLocation: string = "";
```

Comment:
VSCode is complaining that `preferredSourceLocation` is not used. I see it being used in `AbstractGenerateCommand.ts`. Should this variable be used? Or should all entries here be removed?

Reply:
@sghung , I'll take care of this.  It should be used, at least for Swift.